.github/workflows/smoketests.yml

```diff
@@ -46,6 +48,7 @@ jobs:
 
             - name: Cypress run
               # Uses the official Cypress GitHub action https://github.com/cypress-io/github-action
+              if: "!contains(steps.pr_information.outputs.branch_name, 'translation-new-homepage')"
```

Comment:
I guess this condition shouldn't be in a string.

Reply:
I think it's ok. It worked fine as a string in my deriv-experiments repo.